config/settings/housekeeping.q

```diff
@@ -4,3 +4,7 @@
 inputcsv:first .proc.getconfigfile["housekeeping.csv"]
 runtimes:02:00:00
 runnow:0b
+
+\d .windows
+
+version:`w10
```

Comment:
Need to add this to the start of `code/processes/housekeeping.q` (with the other variables):
```
.win.version:@[value;`.win.version;`w7]
```
Not the changed namespace, just had a thought that `.win` already exists so lets continue to use it!